web-ui/view/client_error.ml

```diff
@@ -1,12 +1,89 @@
 open Tyxml.Html
 
+let head =
+  head
+    (title (txt "OCaml-CI - Error"))
+    [
+      meta ~a:[ a_charset "UTF-8" ] ();
+      meta ~a:[ a_http_equiv "X-UA-Compatible"; a_content "IE=edge" ] ();
+      meta
+        ~a:
+          [
+            a_name "viewport"; a_content "width=device-width, initial-scale=1.0";
+          ]
+        ();
+      script ~a:[ a_defer (); a_src "/js/alpine.js" ] (txt "");
+      script ~a:[ a_defer (); a_src "/js/alpine-clipboard.js" ] (txt "");
+      link ~rel:[ `Stylesheet ] ~href:"/fonts/inter.css" ();
+      link ~rel:[ `Stylesheet ] ~href:"/css/main.css" ();
+      link ~rel:[ `Stylesheet ] ~href:"/css/ansi.css" ();
+    ]
+
+let header =
+  header
+    ~a:
+      [
+        a_class
+          [
+            "h-20 flex items-center border-b border-gray-200 bg-white sticky \
+             top-0 z-50";
+          ];
+      ]
+    [
+      div
+        ~a:[ a_class [ "container-fluid flex justify-between items-center" ] ]
+        [
+          a
+            ~a:[ a_href "/" ]
+            [
+              img ~src:"/images/logo.svg" ~alt:"OCaml-CI"
+                ~a:[ a_width 116; a_height 32 ]
+                ();
+            ];
+        ];
+    ]
+
+let body ~code ~reason =
+  body
+    ~a:[ a_class [ "" ] ]
+    [
+      header;
+      div
+        ~a:[ a_class [ "container-fluid py-12 text-center" ] ]
+        [
+          div
+            ~a:
+              [
+                a_class
+                  [
+                    "flex justify-center flex-col items-center fixed left-0 \
+                     top-0 w-full h-full px-10";
+                  ];
+              ]
+            [
+              h1 ~a:[ a_class [ "text-stroke" ] ] [ txt (Fmt.str "%d" code) ];
```

Comment:
Oh cool so this just renders whatever code is passed in. Nice. Did you get a chance to check that this is all ok for 500s as well?

Reply:
Couldn't reproduce one on my side... That's why I wanted to merge it: to make sure it apply.
And yes, this is the error handler given to `Dream` :)